psi4/driver/procrouting/dft_funcs/dict_dh_funcs.py

```diff
@@ -0,0 +1,577 @@
+            "alpha": 0.73
+        }
+    },
+    "c_mp2": {
+        "alpha": 0.27
+    },
+    "citation":
+    '    S. Grimme, J. Chem. Phys., 124, 034108, 2006\n',
+    "description":
+    '    B2PLYP Double Hybrid Exchange-Correlation Functional\n',
+})# yapf: disable
+
+funcs.append({
+    "name": "DSD-BLYP",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.25
+        }
+    },
+    "x_hf": {
+        "alpha": 0.75
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.53
+        }
+    },
+    "c_mp2": {
+        "os": 0.46,
+        "ss": 0.60
+    },
+    "citation":
+    '    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n',
+    "description":
+    '    DSD-BLYP SCS Double Hybrid XC Functional\n',
+})# yapf: disable
+
+funcs.append({
+    "name": "DSD-BLYP-D2",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.29
+        }
+    },
+    "x_hf": {
+        "alpha": 0.71
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.55
+        }
+    },
+    "c_mp2": {
+        "os": 0.46,
+        "ss": 0.43
+    },
+    "citation":
+    '    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n',
+    "description":
+    '    DSD-BLYP Dispersion-corrected SCS Double Hybrid XC Functional (-D2 parameters)\n',
+    "dispersion": {
+        "type": "d2",
+        "params": {
+            "s6": 0.35
+        },
+        "citation":
+        '    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n'
+    },
+})# yapf: disable
+
+funcs.append({
+    "name": "DSD-BLYP-D3BJ",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.29
+        }
+    },
+    "x_hf": {
+        "alpha": 0.71
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.54
+        }
+    },
+    "c_mp2": {
+        "alpha": 1.0,
+        "os": 0.47,
+        "ss": 0.40,
+    },
+    "dispersion": {
+        "type": "d3bj",
+        "params": {
+            "s6": 0.57,
+            "a2": 5.4,
+            "a1": 0.0,
+            "s8": 0.0
+        },
+        "citation": "    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n"
+    },
+    "citation": '    S. Kozuch, J.M.L. Martin, J. Comp. Chem., 34, 2327-2344, 2013\n',
+    "description": '    DSD-BLYP-D3BJ Dispersion-corrected SCS Double Hybrid XC Functional\n',
+})
+
+funcs.append({
+    "name": "CORE-DSD-BLYP",
+    "x_functionals": {
+        "GGA_X_B88": {
+            "alpha": 0.31
+        }
+    },
+    "x_hf": {
+        "alpha": 0.69
+    },
+    "c_functionals": {
+        "GGA_C_LYP": {
+            "alpha": 0.54
+        }
+    },
+    "c_mp2": {
+        "os": 0.46,
+        "ss": 0.37
+    },
+    "citation":
+    '    S. Kozuch, Phys. Chem. Chem. Phys., 13, 20104, 2011\n',
+    "description":
+    '    DSD-BLYP Dispersion-corrected SCS Double Hybrid XC Functional\n' + \
+    '    (full-core parameterization) \n'
```

Comment:
Make this edit to appease the Sphinx. :-)

```
-    '    DSD-BLYP Dispersion-corrected SCS Double Hybrid XC Functional\n' + \
-    '    (full-core parameterization) \n'
+    '    DSD-BLYP Dispersion-corrected SCS Double Hybrid XC Functional (full-core param.)\n'
```

Reply:
This is the only really necessary change, otherwise the docs build will fail. I'm afraid the Sphinx is more particular than I am. :-)